util/log.py

```diff
@@ -4,12 +4,26 @@
 import os
 import six
 import sys
+import time
 
 try:
     from dlstbx.util.colorstreamhandler import ColorStreamHandler
 except ImportError:
     ColorStreamHandler = None
 
+# https://stackoverflow.com/questions/25194864/python-logging-time-since-start-of-program/25196134#25196134
+class ElapsedFormatter:
+    def __init__(self):
+        self.start_time = time.time()
+
+    def format(self, record):
+        elapsed_seconds = record.created - self.start_time
+        elapsed = "{:06.1f}: ".format(elapsed_seconds)
```

Comment:
```suggestion
        elapsed = "{:6.1f}: ".format(elapsed_seconds)
```
suggest dropping leading `0`s.